### PR TITLE
Add tests for creating/getting/deleting tables and rows in raw.

### PR DESCRIPTION
**Pull request by erlendcognite**

### CODE REVIEW

**Comment by erlendcognite** [comment]
Sorry for the messy PR. Accidentally branched off of the previous py2.7 branch. Only thing changed here is the 2 last commits affecting __init__.py and raw.py

File: tests/test_raw.py [hunk 1]
```diff
@@ -38,77 +62,65 @@ def test_delete_databases(self):
...
+        with pytest.raises(APIError) as e:
+            raw.delete_tables(DB_NAME, [TABLE_NAME])
+            # assert re.match("{'code': 404, 'message': 'Did not find any dbs with the given names'}", str(e.value))
+            # assert re.match("{'code': 404, 'message': 'No tables named test_table'}")
+
+
+class TestRows:
+    @pytest.fixture(autouse=True, scope='class')
+    def create_database(self):
+        raw.create_databases([DB_NAME]).to_json()
```
Comment (psalaberria002) [inline tests/test_raw.py]: is to_json necessary?

File: tests/test_raw.py [hunk 2]
```diff
@@ -38,77 +62,65 @@ def test_delete_databases(self):
...
+        raw.create_tables(DB_NAME, [TABLE_NAME]).to_json()
+        yield
+        raw.delete_databases([DB_NAME], recursive=True)
+
+    def test_create_rows(self):
+        response = raw.create_rows(DB_NAME, TABLE_NAME, rows=[RawRowDTO(key=ROW_KEY, columns=ROW_COLUMNS)])
```
Comment (psalaberria002) [inline tests/test_raw.py]: Create rows response is equals to empty array?

Comment (erlendcognite) [reply tests/test_raw.py]: yes

**Pull request merged**